sites/fast-website/src/app/components/feature-card/index.ts

```diff
@@ -0,0 +1,14 @@
+import { customElement } from "@microsoft/fast-element";
+import { FeatureCardTemplate as template } from "./feature-card.template";
+import { FeatureCardStyles as styles } from "./feature-card.styles";
+import { FASTCard } from "@microsoft/fast-components";
+
+@customElement({
+    name: "feature-card",
```

Comment:
could site be added to this name? such as site-feature-card

Reply:
Done